Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
-        int digitCount = content.Count(char.IsDigit);   
+        var line = reader.ReadLine()!;
 
-        Console.WriteLine($"The file contains {letterCount} letters and {digitCount} digits.");
-    }
-    else
-    {
-        Console.WriteLine($"File not found at path: {filePath}");
-    }
+        foreach (var character in line)
+        {
+            var helper = new Helper();
+
+            var isCharacter = helper.IsCharacterAndIfINeedToCount(character);
+            var isDigit = helper.IsDigitAndIfItIsInTheList(character);
+
+            IncreaseLetter_CountIfTrue(letter_count, isCharacter);
+            helper.IncreaseDigitCount(DigitCount, isDigit);
+        }
+        
+        void IncreaseLetter_CountIfTrue(int letter_countt, bool isCharacter)
```

Comment:
Jei jau darome IncreaseDigitCount Helper klaseje, tada ir sia funkcija kisame po Helper klase
Be to, kadangi kitur funkcijos uzvadintos kaip PascalCase tada ir sia funkcija geriau vadiname taip pat - IncreaseLetterCount